query-engine/core/src/query_graph_builder/read/utils.rs

```diff
@@ -254,14 +255,23 @@ pub(crate) fn get_relation_load_strategy(
     requested_strategy: Option<RelationLoadStrategy>,
     cursor: Option<&SelectionResult>,
     distinct: Option<&FieldSelection>,
+    order_by: &[OrderBy],
```

Comment:
Passing `QueryArguments` here could simplify code a bit but then we'll have to store `QueryArguments` in `RecordQuery` for findUnique rather than just a model and filters (or build query arguments temporarily but that would involve cloning the filters).

Reply:
Actually it won't be necessary after https://github.com/prisma/team-orm/issues/954 so it's better the way it is.